packages/tinytex/install

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -x
+set -e
+
+apt-get update -qq
+
+# install inkscape to use includesvg of LaTeX's svg package
+apt-get install -y inkscape
```

Comment:
Is there a more targeted dependency that can be added here? It seems odd to install `inkspace` in a headless environment.

Reply:
So I looked through package repositories and `Dockerfile`s and could not find headless versions. Even Alpine only has a package `inkscape` which can be used for GUI applications as well (e.g. see [minidocks/inkscape](https://github.com/minidocks/inkscape)).